content/en/docs/automatic/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Automatic & Assisted Instrumentation
+aliases: [/docs/zero-code]
+redirects:
+  - { from: /docs/instrumentation/*, to: ':splat' } # Only for `en`
+  - { from: 'net/*', to: 'dotnet/:splat' }
+weight: 260
+---
+
+Besides **code-based** instrumentation using the
+[language-specific APIs and SDKs](/docs/languages/) the OpenTelemetry project
+provides a set of solutions for **automatic** (or **zero-code**) and
```

Comment:
We might need to be even more precise in the wording. Do we have definitions of the terms somewhere?

Reply:
We have them introduced in the concept page, can link it here once again, I also updated that page with the PR, see 

https://github.com/open-telemetry/opentelemetry.io/pull/7157/files#diff-d7365fbf3339a2546966ef878b1b7af558848fc1abad152c217830c3ab127c8a

Does this work?